akka-projection-jdbc/src/main/scala/akka/projection/jdbc/scaladsl/JdbcProjection.scala

```diff
@@ -73,6 +72,10 @@ object JdbcProjection {
    * This means that if the projection is restarted from previously stored offset then some elements may be processed
    * more than once.
    *
+   * The [[JdbcHandler.process()]] in [[handler]] will be wrapped in a transaction, it is highly recommended to use
```

Comment:
```suggestion
   * The [[JdbcHandler.process()]] in [[handler]] will be wrapped in a transaction. It is highly recommended to use
```

Minor typo fix. 

On the other hand, although we can tell users that `process` will participate on a transaction, it doesn't make much difference for when using it with `AtLeastOnce` strategy. Only the user's code will be added to the transaction.

Reply:
This is slightly different than the DBIO case. The user can't combine monads here, it's one method call, one transaction. See my comment about DBIO later here.

Reply:
> On the other hand, although we can tell users that process will participate on a transaction, it doesn't make much difference when using it with AtLeastOnce strategy. Only the user's code will be added to the transaction.

Yeah, but just telling the users the library will invoke the `commit` and handle the `rollback` (even if they are not using `exactly-once`) is useful.